src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1936,11 +1936,6 @@ private async Task<Organization> GetOrgById(Guid id)
 
     private static void ValidatePlan(Models.StaticStore.Plan plan, int additionalSeats, string productType)
     {
-        if (plan is not { LegacyYear: null })
```

Comment:
may we should change or remove the LegacyYear in the PasswordManagerPlanStore instead of remove the check

Reply:
Yeah, this is an excellent discussion to bring up. Other than this bit of removed validation, the only place I've seen `LegacyYear` used is over on the client side at [plan.response.ts#L40](https://github.com/bitwarden/clients/blob/master/libs/common/src/billing/models/response/plan.response.ts#L40). The uses of the property are limited to organization-plans.component.ts on lines [211](https://github.com/bitwarden/clients/blob/master/apps/web/src/app/billing/settings/organization-plans.component.ts#L211) and [232](https://github.com/bitwarden/clients/blob/master/apps/web/src/app/billing/settings/organization-plans.component.ts#L232). The UI only uses that property to determine which plans it should show to the user when selecting a plan to create the org with or to upgrade to.

From what I can see, the `disabled` flag accomplishes the same thing as `legacyYear`, so it's possible that we could go ahead with deleting this property entirely in favor of setting the legacy plans as disabled. What would you guys think about that?